### PR TITLE
Minor schedule layout improvements

### PR DESCRIPTION
**Pull request by argyleink**
- feature phone session desc doesnt touch right edge
- small changes to the fluidity and adaption break point for shared day schedule space

gif showing changes

schedule-layout-nits

### CODE REVIEW

**Comment by jakearchibald** [comment]
Are you happy with the line lengths at max width?

File: src/schedule/style.css [hunk 1]
```diff
@@ -177,6 +177,7 @@
 
 .item {
   grid-column: last;
+  overflow: hidden;
```
Comment (jakearchibald) [inline src/schedule/style.css]: This means we lose the shadow and speech bubble. Intentional?

**Comment by argyleink** [comment]
right, those line lengths are a bit over the optimal thresholds, but, my eye would rather they complete the layout grid than not. your thoughts there? is the wide paragraph size distracting?

File: src/schedule/style.css (hunk 1 above)
Comment (argyleink) [reply src/schedule/style.css]: nice, lemme go review! i hadnt noticed that side effect

**Pull request merged**